test/deprecated.jl

```diff
@@ -1,10 +1,12 @@
 module TestDeprecated
 
-using Test, DataFrames, Random
+using Test, DataFrames, Random, Logging
 import DataFrames: identifier
 
 const ≅ = isequal
 
+old_logger = global_logger(NullLogger())
```

Comment:
Is this really needed in this file (it's not modified AFAICT)?

Reply:
What do you mean by "it's not modified"?

This line disables outputting deprecation warnings from deprecated.jl.
There are hundreds of them, and they are expected.
I do not think anyone is able to digest them so I disable printing them (if something errors we will catch this as then CI will fail).

The reason of these changes is that after them it is much easier to find real problems in the actual testing code (as if you get an error or a warning it is easy to locate it by searching the logs). Does it make sense (it for sure makes my life easier)?